apps/web/lib/services/blockchain/__tests__/docker-integration.test.ts

```
@@ -0,0 +1,444 @@
+
+  if (segments[0] === 'progress-tracker') {
+    return mockProgressTrackerEndpoint(segments.slice(1), method, body);
+  } else if (segments[0] === 'badge-tracker') {
+    return mockBadgeTrackerEndpoint(segments.slice(1), method, body);
+  } else if (segments[0] === 'auth-controller') {
+    return mockAuthControllerEndpoint(segments.slice(1), method, body);
+  } else if (segments[0] === 'graduation-nft') {
+    return mockGraduationNFTEndpoint(segments.slice(1), method, body);
+  } else if (segments[0] === 'system') {
+    return mockSystemEndpoint(segments.slice(1), method, body);
+  }
```

Biome (2.1.2)

[error] This else clause can be omitted because previous branches break early.

Safe fix: Omit the else clause.


(lint/style/noUselessElse)